filesystem/fat32/fat32.go

```diff
@@ -241,10 +252,11 @@ func Create(f util.File, size int64, start int64, blocksize int64, volumeLabel s
 		return nil, fmt.Errorf("Could not create a valid byte stream for a FAT32 Filesystem Information Sector: %v", err)
 	}
 	fsisPrimary := int64(fsisPrimarySector * uint16(SectorSize512))
-	fsisBackup := int64(fsisBackupSector * uint16(SectorSize512))
 
 	f.WriteAt(fsisBytes, fsisPrimary+int64(start))
-	f.WriteAt(fsisBytes, fsisBackup+int64(start))
+	if fsisBackupSector > 0 {
+		f.WriteAt(fsisBytes, int64(fsisBackupSector+1)*int64(SectorSize512)+int64(start))
```

Comment:
Here we write it again, with the FSIS bytes, but we write it one sector _after_ the FSIS backup sector?

Reply:
It looks like this is actually the backup boot sector
> For FAT32 file systems, the reserved sectors include a File System Information Sector at logical sector 1 and a Backup Boot Sector at logical sector 6.
see e.g. https://en.wikipedia.org/wiki/Design_of_the_FAT_file_system

I have to double check if the FSIS is added there, but I have seen references that 3 backup blocks are written, which would be the FSIS on change (here) and above

Reply:
Ah, later on the same wikipedia page
> First logical sector number of a copy of the three FAT32 boot sectors, typically 6.

So it's not a backup of the FSIS but of the whole boot sector

Reply:
Should the struct member be renamed, too?